### ISSUE TITLE
Apple Silicon architecture support

### ISSUE THREAD

**Issue by andrei-dascalu** [cosmtrek/air#143]
Hello,

When trying to use `air` with my (pun inevitable) `macbook air` using the new M1 chip, I'm getting the following error:

```
cosmtrek/air crit platform linux/arm64 is not supported.  Make sure this script is up-to-date and file request at https://github.com/cosmtrek/air/issues/new
```

I'm not sure what it would take to have a compatible install,  as from what I understand M1 compatibility would make `air` >= 1.15

**Comment by nickbryan** [comment]
Yeah, I am also experiencing compatibility issues with Go 1.16. `package embed is not in GOROOT`. Would be ace to get this updated to 1.16

**Comment by nickbryan** [comment]
I think this is fixed in https://github.com/cosmtrek/air/pull/144 we just need a release :)

**Comment by andrei-dascalu** [comment]
Awesome!

**Comment by coolnalu** [comment]
Followed this article and fixed the issue. https://kecci.medium.com/docker-comstrek-air-not-supported-linux-arm64-in-golang-with-macbook-m1-solved-f7997bd54ee

**Comment by ostap1010** [comment]
Hello! Can anyone please help. I am using cosmtrek/air docker image with Apple M1 and it always runs in "amd64" emulation. Is there a possibility to run it without emulation? (Seems like some functions are not working)
